### PR TITLE
chore: use local linter for terraform-google-module-template

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by apeabody** [comment]
Instead of deleting the linter we should remove from tf state

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -391,9 +391,10 @@ locals {
       topics      = local.common_topics.db
     },
     {
-      name        = "terraform-google-module-template"
-      org         = "terraform-google-modules"
-      description = "Provides a template for creating a Cloud Foundation Toolkit Terraform module"
+      name              = "terraform-google-module-template"
+      org               = "terraform-google-modules"
+      description       = "Provides a template for creating a Cloud Foundation Toolkit Terraform module"
+      disable_lint_yaml = false
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:397]: Isn't this going to add the lint yaml to the repo when it's already there?

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:397]: Yup, wrong `bool` value :)

**Comment by apeabody** [comment]
applied with manual state adjustment

**Pull request merged**